client/src/components/Home.js

```diff
@@ -1,13 +1,43 @@
 import React from "react";
 import "../styles/Home.css";
 import { Header } from "semantic-ui-react";
+import { Menu } from "semantic-ui-react";
 
 const Home = () => {
   return (
     <div>
       <header className="home-header">
```

Comment:
You imported Menu from Semantic UI but I don't see it you used in this file.
Also, there is Header that is not used. You can to use Semantic UI for Header, and any other html tags in your file.
You can have a look how it should be used here https://react.semantic-ui.com/elements/header/

Reply:
yes , I have deleted the Menu from the code.
the Header now has been used now in the header tag and logo,but the font color of the header changed to black in opposite to the required wire-frame:( so how can I return it to white?